context.go

```diff
@@ -385,7 +385,10 @@ func (c *context) String(code int, s string) (err error) {
 }
 
 func (c *context) JSON(code int, i interface{}) (err error) {
-	if c.echo.Debug {
+	query := c.request.URL.Query()
```

Comment:
You can directly use `_, pretty := c.QueryParams()["pretty"]`. Just for consistency please rename the `isPretty` to `pretty`.

PS: Same for XML